docs/modules/operation/pages/device-config-backup/configuration.adoc

```diff
@@ -1,35 +1,35 @@
 == Poller configuration
```

Comment:
= Poller configuration

Title level needs to be updated, as the table of contents did not build properly.